lib/config/presets/index.ts

```diff
@@ -283,6 +287,7 @@ export async function resolveConfigPresets(
     'resolveConfigPresets'
   );
   let config: AllConfig = {};
+  const tempExtends: string[] = [];
```

Comment:
```suggestion
  const unresolvedPresets: string[] = [];
```

Try to avoid naming things "temp" whenever possible. Is this name a more accurate description?

Reply:
Yes and got it 👍